npm install karma issues

While installing this I get the following error:

 ERR! peerinvalid The package karma does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer karma-script-launcher@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-chrome-launcher@0.1.4 wants karma@>=0.9.3
npm ERR! peerinvalid Peer karma-firefox-launcher@0.1.3 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-html2js-preprocessor@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-coffee-preprocessor@0.1.3 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-requirejs@0.2.2 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-phantomjs-launcher@0.1.4 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-junit-reporter@0.2.2 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-mocha@0.1.4 wants karma@>=0.12.8
npm ERR! peerinvalid Peer karma-chai@0.0.2 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-jasmine@0.1.5 wants karma@>=0.9

npm ERR! System Darwin 13.2.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/***_/Code/angular-client-side-auth
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.15
npm ERR! code EPEERINVALID
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/**_*/Code/angular-client-side-auth/npm-debug.log
npm ERR! not ok code 0

my steps were
1. git clone into this repo
2. run npm install
3. run into this error.

I have tried:
1. modifying the package.json
2. reinstalling karma (globally and locally..)
3. reinstalling bower
4. npm update
5. bower update

I'd consider myself a beginner, and I do not know what to try next. Can you help me?

--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
     "requirejs": "~2.1.10",
     "karma-requirejs": "~0.2.1",
     "karma-phantomjs-launcher": "~0.1.2",
-    "karma": "~0.10.9",
+    "karma": "~0.12.16",
     "karma-junit-reporter": "~0.2.1",
     "karma-mocha": "~0.1.1",
     "karma-chai": "0.0.2",

